tests/TestCase/View/ViewTest.php

```diff
@@ -897,9 +897,10 @@ public function testPrefixElement()
     }
 
     /**
-     * Test elementInexistent method
+     * Test loading inexistent view element
      *
      * @expectedException \Cake\View\Exception\MissingElementException
+     * @expectedExceptionMessage Element file "Element\non_existent_element.ctp" is missing
```

Comment:
On *nix the path will have forward slash not backslash. So instead of the annotations you ca use use phphunit's methods for checking exception class and message.